cmd/lint/main.go

```diff
@@ -24,7 +24,7 @@ var commandRules = []linter.CommandRule{
 
 	linter.Filter(linter.RequireCapitalizeProperNouns("Short", properNouns), linter.ExcludeCommand("local current")),
 	linter.RequireEndWithPunctuation("Short", false),
-	linter.Filter(linter.RequireNotTitleCase("Short", properNouns)),
+	linter.Filter(linter.RequireNotTitleCase("Short", properNouns), linter.ExcludeCommand("schema-registry kek"), linter.ExcludeCommand("schema-registry dek")),
```

Comment:
Can you add KEK and DEK to the list of proper nouns instead?

Reply:
they have been added. This is for "don't title case short description"

Reply:
Oh, in that case I think "Key Encryption Key" is what's being identified as title case, so we should add that string to the list of proper nouns